frontend-react/src/hooks/api/UseActionHistory/UseActionHistory.ts

```diff
@@ -0,0 +1,73 @@
+import { useSuspenseQuery } from "@tanstack/react-query";
```

Comment:
Unless I'm mistaken, `useActionHistory` isn't used anywhere? This is setting up for a future PR? If so, can we add some comments?

Reply:
Aside from the existing hooks that were reorganized, all of these are unused. This is scaffolding by getting the bulk of remaining ones into the project to quickly iterate on more nuanced PRs to use them in pages.

Reply:
Still might be good to at least add a comment on each denoting that?